tests/integration/new_relations/test_new_relations.py

```diff
@@ -41,15 +41,15 @@
 
 @pytest.mark.abort_on_fail
 async def test_database_relation_with_charm_libraries(
-    ops_test: OpsTest, application_charm, database_charm
+    ops_test: OpsTest, postgrsql_test_app_charm, database_charm
```

Comment:
The same minor change in other functions signatures.
```suggestion
    ops_test: OpsTest, postgresql_test_app_charm, database_charm
```